mod/wikirate/set/right/contributed_analysis.rb

```diff
@@ -1,11 +1,15 @@
 format :html do
   def default_header_args args
-    args[:count] = Card.search :type_id=>WikirateAnalysisID,
-                    :right_plus=>[ 'article', {:or=>{:created_by=>card.left.name, :edited_by=>card.left.name }}],
-                    :return=>:count
     args[:icon] = nest(Card.fetch('venn icon'), :view=>:core, :size=>:icon)
   end
 
+  def contribution_count
+    @cc ||= Card.search :type_id=>WikirateAnalysisID,
+              :right_plus=>[ 'article', {:or=>{:created_by=>card.left.name, :edited_by=>card.left.name }}],
+              :return=>:count
+  end
+
+
```

Comment:
:edited_by includes the initial edit (creation).  So we don't really need both created_by and edited_by.

I looked at this query a bit. Looks like it might be possible to speed it up a bit, but I'm not sure what the systemic effects would be.